app/core/Tracker.coffee

```diff
@@ -236,18 +236,18 @@ module.exports = class Tracker extends CocoClass
     return if me.isAdmin() or @shouldBlockAllTracking()
     return unless me.isTeacher(true)
     loadSegmentIo()
-    .then =>
-      @segmentLoaded = true
+    .then (segmentIo) =>
```

Comment:
I'm not completely happy with the name `segmentIo`.
This returns a list of segment analytics methods if it passes.